packages/terra-date-picker/tests/jest/__snapshots__/DatePicker.test.jsx.snap

```diff
@@ -2,11 +2,10 @@ exports[`test should render a date picker with disabled dates 1`] = `
 <ResponsiveElement
   className="terra-DatePicker"
   defaultElement={
-    <DatePicker
+    <t
```

Comment:
I had to update the version of react-datepicker from 0.44 to 0.46. I'll double check, it does look weird